src/GraphQL/Transport/IGraphQLSerializer.cs

```diff
@@ -28,5 +28,11 @@ public interface IGraphQLSerializer
         /// A <paramref name="value"/> of <see langword="null"/> returns <see langword="default"/>.
         /// </summary>
         T? ReadNode<T>(object? value);
+
+        /// <summary>
+        /// Indicates whether this serializer makes asynchronous calls to the underlying stream
+        /// while serializing or deserializing.
```

Comment:
```suggestion
        /// while serializing or deserializing. This property is auxiliary API in nature and may
        /// help to avoid an additional/unnecessary buffering at caller side.
```